### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by githubuser45566**

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -157,7 +157,7 @@ id,name,alt_names,network,owners,country,subdivision,city,broadcast_area,languag
 312TV.kg,312 TV,312 ТВ;312 Сериал,312 TV,TUZ.KG,KG,,Bishkek,c/KG,rus;kir,entertainment,FALSE,,,,http://312tv.kg,https://i.ibb.co/S6sZvqk/iLSGi6c.png
 313Digital.sn,313 Digital,,,,SN,,Dakar,c/SN,fra,news,FALSE,,,,,https://i.imgur.com/qplFaTa.jpg
 31Kanal.kz,31 Kanal,31 канал,,,KZ,,Almaty,c/KZ,kaz,general,FALSE,1993-04-12,,,https://31.kz/,https://i.imgur.com/5Bp9ljw.png
-324.es,3/24,,,,ES,ES-CT,Sant Joan Despi,c/ES;c/AD,cat,news,FALSE,2003-01-01,,,https://www.ccma.cat/tv3/directe/324/,https://i.imgur.com/l6NzX2t.png
+324.es,24-Mar,,,,ES,ES-CT,Sant Joan Despi,c/ES;c/AD,cat,news,FALSE,2003-01-01,,,https://www.ccma.cat/tv3/directe/324/,https://i.imgur.com/l6NzX2t.png
```
Comment (githubuser45566) [inline data/channels.csv]: This was some formatting issue, fixed in another pull request (Still learning)

Comment (BellezaEmporium) [reply data/channels.csv]: Actually, not a formatting issue. This is the name of the channel you're modifying, and the channel is *actually named* 3/24.

File: data/languages.csv [hunk 2]
```diff
@@ -7891,4 +7891,5 @@ zyj,Youjiang Zhuang
 zyn,Yongnan Zhuang
 zyp,Zyphe Chin
 zza,Dimili
-zzj,Zuojiang Zhuang
\ No newline at end of file
+zzj,Zuojiang Zhuang
+pun,Punjabi
```
Comment (BellezaEmporium) [inline data/languages.csv:7895]: Punjabi already exists.

pan | Panjabi

File: data/channels.csv (hunk 1 above)
Comment (githubuser45566) [reply data/channels.csv]: Microsoft Excel detects this as date and changes automatically every time I open the file 😞

**Review by Alstruit — CHANGES_REQUESTED** [review]
Punjabi language already exists under pan.

**Pull request closed without merging**